custom_components/tuya_local/devices/etop_fcu_thermostat.yaml

```diff
@@ -0,0 +1,141 @@
+name: ETOP-FCU(CF) thermostat
```

Comment:
The name here should be without branding, in case of other matching devices.  The branding should be moved into the name under the products section.

Reply:
Updated.